pyrtl/helperfuncs.py

```diff
@@ -1683,3 +1683,27 @@ def __len__(self):
             return len(self._components)
 
     return _WireMatrix
+
+def one_hot_to_binary(w) -> WireVector:
+    '''Takes a one-hot input and returns the bit position of the high bit in binary.
+
+    :param w: WireVector or a WireVector-like object or something that can be converted
+        into a Const (in accordance with the as_wires() required input).
+    :return: The bit position of the high bit in binary as a WireVector.
+
+    If the input contains multiple 1s, the bit position of the first 1 will
+        be returned. If the input contains no 1s, 0 will be returned. 
+    '''
+
+    if not isinstance(w, WireVector):
```

Comment:
I don't think this check is needed since `as_wires` should pass through `w` if it's already a `WireVector`